scripts/build-config-fixtures.js

```diff
@@ -0,0 +1,52 @@
+import yazl from 'yazl'
+import fs from 'node:fs/promises'
+import { createWriteStream } from 'node:fs'
+
+const CONFIG_FIXTURES_PATH = new URL(
+  '../tests/fixtures/config',
+  import.meta.url
+).pathname
+const dir = await fs.readdir(CONFIG_FIXTURES_PATH)
```

Comment:
You could instead pass the `withFileTypes` option, which will give back an array of `fs.Dirent` objects instead of strings. Then you don't need to call `fs.stat` below, because you can check if it's a directory right there.

Take it or leave it!

Reply:
done!